general/development/policies/accessibility.md

```diff
@@ -93,6 +93,10 @@ An overview of Moodle LMS' conformance with the [WCAG 2.1](https://www.w3.org/TR
 
 An overview of Moodle Workplace's conformance with the [WCAG 2.1](https://www.w3.org/TR/WCAG21/) guidelines can be found in our [accessibility conformance report](https://docs.moodle.org/en/Moodle_Workplace_VPAT#WCAG_2.x_report).
 
+#### Moodle App
+
+An overview of Moodle App's conformance with the [WCAG 2.2](https://www.w3.org/TR/WCAG22/) guidelines can be found in our [accessibility conformance report](https://docs.moodle.org/en/Moodle_app_VPAT#WCAG_2.x_report).
```

Comment:
I realise that we'll have three links with an accessible name of `accessibility conformance report` that point to different URLs, and this is an accessibility issue. Let's change
> can be found in our [accessibility conformance report]

to

> can be found in the [Moodle App accessibility conformance report]

I'll put up a separate pull request to fix LMS and Workplace.